projects/ad7768-evb/ad7768_evb.mk

```diff
@@ -1,20 +1,20 @@
-## NOOS-DIR:
-##		is the no-OS root variable (do not override)
-##
-## M_INC_DIRS:
-##		list all your include folders (all *.h and *.c files from these folders are included
-##		in compile and passed as directives)
-##
-## M_HDR_FILES:
-##		if you want to hand pick files, use this variable to list header files.
-##
-## M_SRC_FILES:
-##		if you want to hand pick files, use this variable to list source files.
-
-M_INC_DIRS +=  $(NOOS-DIR)/ad7768-evb
-M_INC_DIRS +=  $(NOOS-DIR)/drivers/adc/ad7768
-
-M_HDR_FILES :=
-
-M_SRC_FILES :=
-
+## NOOS-DIR:
```

Comment:
Nitpick: This seems just a rename, maybe you modified the line endings

Reply:
I did not mean to change anything in this file when moving, true. Will fix.